Backend/index.js

```diff
@@ -48,20 +48,48 @@ async function triggerBuild(buildUrl, options) {
   return response;
 }
 
+//For single row query to get additional information
+app.post('/api/history', (req, res) => {
+  const buildId = req.body.buildId;
+  db.query(`SELECT * FROM BUILDS WHERE build_id = ${buildId}`, function (err, result, fields) {
+    if (err) {
+      console.log(fields);
+      throw err;
+    } 
+    res.status(200);
+    res.json(result);
+  });
+});
+
+
+//Returns wholde table. Could be narrowed down to needed.
+app.get('/api/history', (req, res) => {
+  db.query('SELECT * FROM BUILDS', function (err, result, fields) {
+  if (err) {
+    console.log(fields);
```

Comment:
Same here

Reply:
Yeah, these were for eslint to "use" unused variable.  Others are for actual informational purpose - not for testing. I can ofc remove them.

Reply:
I think the right solution would be to have a some way to only have the wanted parameters in the function call. If tools, we are using to ensure quality, require as to make a bad solutions, then we have issue with the tool. In many other places the unused parameter check is very usefully so I would rather not remove it from eslint. Was there way to ignore eslint rule in spesific spot in code?